gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,8 +1,8 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 # checksum was taken from https://gradle.org/release-checksums
-distributionSha256Sum=1b6b558be93f29438d3df94b7dfee02e794b94d9aca4611a92cdb79b6b88e909
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.0.1-bin.zip
+distributionSha256Sum=ff7bf6a86f09b9b2c40bb8f48b25fc19cf2b2664fd1d220cd7ab833ec758d0d7
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.0.2-bin.zip
```

Comment:
nit: should we use `gradle-8.0.2-all.zip` instead of `gradle-8.0.2-bin.zip`?

Reply:
imo most devs dont need gradle docs and source code locally since they are not working on the build logic
also in CI this means downloading a smaller file

but open to hear what others say